src/components/avatar/Avatar.tsx

```diff
@@ -19,21 +21,29 @@ export interface AvatarProps {
     className?: string;
     /** Users id */
     id?: string | number | null;
+    /** Whether this avatar should be labeled as external in the current context */
+    isExternal?: boolean;
     /**
      * Users full name.
      *
      * Required if "avatarUrl" is not specified.
      */
     name?: string | null;
+    /** Show the external avatar marker if the avatar is marked as for an external user */
+    shouldShowExternal?: boolean;
```

Comment:
is shouldShowExternal the feature flip? shouldn't we always just show the external badge if it's an external user?

Reply:
yes we should, tho some interfaces can reveal this information in unique ways, and others may already include a badge that needs to be removed